contracts/bridge/BridgeFee.sol

```diff
@@ -19,15 +19,29 @@ contract BridgeFee {
         receiver = _receiver;
     }
 
-    function _payKLAYFee(uint256 _fee) internal {
-        if(isValidReceiver() == true && _fee > 0) {
-            receiver.transfer(_fee);
+    function _payKLAYFeeAndRefundChange(uint256 _feeLimit) internal {
+        uint256 fee = feeOfKLAY;
+
+        if(isValidReceiver() == true && fee > 0) {
+            require(_feeLimit >= fee, "invalid feeLimit");
+
+            receiver.transfer(fee);
+            msg.sender.transfer(_feeLimit.sub(fee));
+        } else {
+            msg.sender.transfer(_feeLimit);
```

Comment:
How about simply reverting the tx?

Reply:
if isValidReceiver() == falise, it means there is no receiver. So in this case, it should work with zero fee.